app/code/core/Mage/Core/Model/Observer.php

```diff
@@ -82,7 +77,7 @@ public function addSynchronizeNotification(Varien_Event_Observer $observer)
      */
     public function cleanCache(Mage_Cron_Model_Schedule $schedule)
     {
-        Mage::app()->getCache()->clean(Zend_Cache::CLEANING_MODE_OLD);
+        Mage::app()->getCache()->clean();
```

Comment:
This method is called by cron every hour, and it should clean outdated cache entries, but this is flushing the entire cache, right?

I think we need the following in `Mage_Core_Model_Cache`:
```php
    public function prune(): bool
    {
        if ($this->_frontend implements \Symfony\Component\Cache\PruneableInterface) {
            return $this->_frontend->prune();
        }
        return true;
    }
```

And call `Mage::app()->getCache()->prune();` instead, but I haven't tested this yet.

Reply:
absolutely, done in https://github.com/MahoCommerce/maho/pull/165/commits/e13d094d4371bafc9fdcf7d2e3860d0cf91a6c7a